tests/system-tests/ipsec/README.md

```diff
@@ -0,0 +1,228 @@
+# Telco Verification IPSec System Test Troubleshooting
+
+## Overview
+This document is intended to be a brief troubleshooting guide if there are problems
+with the Telco Verification IPSec System Tests. Normally, if these tests fail, it
+will be because the IPSec tunnel is not correctly established.
+
+IPSec tunnels will have 2 endpoints, and in the context of this testing, one endpoint
+will be the SNO cluster, and the other end is usually a Security Gateway. For these
+test cases, the Security Gateway is usually a simple RHEL VM running in the lab.
+
+## Telco Verification IPSec System Test cases
+Currently, for the SNO use case, these are the test cases:
+
+* ipsec_tunnel_deployment.go - verify the tunnel is established after deployment
+* ipsec_packets_snoegress.go - verify packets egress the cluster via the IPSec tunnel
+* ipsec_packets_snoingress.go - verify packets ingress the cluster via the IPSec tunnel
+
+### Troubleshooting ipsec_tunnel_deployment test failure
+This will most likely be the most common failure and will most likely fail because
+the tunnel is not established.
+
+If the tunnel is not established, it will most likely be for one of the following
+reasons:
+
+1. The Security Gateway is down, either the server or the tunnel on the server
+2. There is no connectivity between the SNO cluster and the Security Gateway 
+3. The IP addresses have changed either on the SNO cluster or the Security Gateway
+
+Find details on how to check these in the following sections.
+
+#### Checking the Security Gateway
+
+The remote end of the SNO IPSec tunnel is often-times called a Security Gateway.
+Currently the Security Gateway is running on a RHEL VM called `secgw.kni-qe-0.lab.eng.rdu2.redhat.com`,
```

Comment:
Could you please update `kni-qe-0.lab.eng.rdu2.redhat.com` with a generic domain name such as  `example.com`?